exporter/datadogexporter/config.go

```diff
@@ -54,20 +55,21 @@ func (api *APIConfig) GetCensoredKey() string {
 
 // DogStatsDConfig defines the DogStatsd related configuration
 type DogStatsDConfig struct {
+	// FIXME Use confignet.NetAddr
```

Comment:
This is not trivial and since we are going to remove this mode I decided to just add a note.